### PR TITLE
feat: Add Finnish (v1.1) 

### LINKED ISSUE #1332: Site Translation: Add Finnish (v1.1)
### Prize Bounty
40 DAI

### Challenge Description
Finnish translations have completed review & are ready to be added to the website!

### Submission Requirements
1. Download the [files from CrowdIn](https://crowdin.com/project/ethereumfoundation/fi#) for this version of the content (`Contents v.1.1`) - here's a [screenshot showing how](https://share.getcloudapp.com/ApujZjyn). You may need to sign up for Crowdin & join our project in order to download the files - here's the [invite link](https://crowdin.com/project/ethereumfoundation/invite).

2. Create new directory: /src/content/translations/fi/

3. Copy in the v1.1 Crowdin files:
- Copy all files into the /src/content/translations/fi/ directory, keeping the same directory structure as CrowdIn (i.e. each page is a subdirectory with an index.md filename, a strings.json at the translation root & an index.md at the root of the directory)
- Copy the strings.json file into the /src/intl/ directory & rename it to fi.json (i.e. /src/intl/fi.json), replacing the existing file
- Delete the root index.md file (i.e. /src/content/translations/fi/index.md)
	- Beforehand: within the index.md file, move the `title` value & `og:description`'s `content` property into the fi.json file from the previous step. Use the keys `"page-home-meta-title"` & `"page-home-meta-description"` as keys for these values. (See [en.json](https://github.com/ethereum/ethereum-org-website/blob/dev/src/intl/en.json#L95) as an example for how this should look)
- Delete the /build/ directory (i.e. /src/content/translations/fi/build/)
	- Beforehand: within the index.md file of /build/, move the `title` value & `og:description`'s `content` property into the fi.json file from the previous step. Use the keys `"page-build-meta-title"` & `"page-build-meta-description"` as keys for these values. (See [en.json](https://github.com/ethereum/ethereum-org-website/blob/dev/src/intl/en.json#L53) as an example for how this should look)

4. Within each newly added markdown file...
- Ensure each file is valid markdown (sometimes Crowdin throws in some wacky HTML elements, like a `<br>` or a `<em>`, that breaks the site build)
- Update the frontmatter:
    - Update the `lang` property (it should be `fi`)
    - Remove any `og:title` or `og:description` properties - replace those contents with `title` & `description` field names
    - Update all `sidebar` fields from `auto` to `true`
    - Increase all `sidebarDepth` fields by 1 (e.g. 0 --> 1 & 1 --> 2)
    - (It might help to [use this commit as a reference](https://github.com/ethereum/ethereum-org-website/pull/1221/commits/bc1f62929147eb32df981a9d1a784f04636f2775))

5. Update all internal links of the new markdown files to include the language path by running our [script](https://github.com/ethereum/ethereum-org-website/blob/dev/scripts/updateInternalLinks.js), i.e. run `yarn update-internal-links fi` from the root of the project.

6. Add an object to [the languageMetadata object within /src/utils/translations.js](https://github.com/ethereum/ethereum-org-website/blob/dev/src/utils/translations.js#L5):
```
fi: {
    version: 1.1,
    language: "Suomalainen",
    "language-english": "Finnish",
  },
```

**Additional context**

For further guidance, see https://github.com/ethereum/ethereum-org-website/pull/1223 as a recent translation addition.

### Judging Criteria
The prize will be awarded to the first valid submission.

### PR DESCRIPTION
**Pull request by aranhaagency**
## Description
Finnish translation added

## Related Issue
close #1332

### CODE REVIEW

File: src/content/translations/fi/dapps/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Käytä Ethereumin sovelluksia (dapps)
+meta:
```
Comment (samajammin) [inline src/content/translations/fi/dapps/index.md:3]: Same issues here as in your other PR (#1335) - please take note for next time.

**Comment by samajammin** [comment]
@aranhaagency did you / can you submit your work on the bounty so I can pay it out? https://gitcoin.co/issue/ethereum/ethereum-org-website/1332/100023522

**Comment by aranhaagency** [comment]
@samajammin I tried to send it on gitcoin but I think this is with a bug or rule that only a user can work on the bounty

**Comment by octavioamu** [comment]
@aranhaagency is not a but, as it is a traditional bounty just one person can be working at each time. Until the other person stop working nobody else can apply to work.

**Pull request merged**